api/v1alpha1/gitrepository_types.go

```diff
@@ -62,6 +66,16 @@ type GitRepositoryRef struct {
 	Commit string `json:"commit"`
 }
 
+// GitRepositoryVerification defines the GPG signature verification process
+type GitRepositoryVerification struct {
+	// Mode describes what git object should be verified.
```

Comment:
```suggestion
	// Mode describes what git object should be verified, currently ('head').
```

To match the documenting format of enums in the spec.